lib/homepage.jsx

```diff
@@ -38,11 +39,10 @@ var CaseStudies = React.createClass({
   render: function() {
     return (
       <div className="case-studies">
-        <blockquote className="primary-quote">
-          <img src="/img/maurya-nyc@2x.png" alt="Maurya NYC Quote" width="148" />
-          Web literacy is about more than coding - it is about how you can be a better web citizen.
-          <small>Maurya C. New York, United States</small>
-        </blockquote>
+        <Blockquote className="primary-quote"
```

Comment:
Cool, I like the idea of componentizing this into a thing we can easily reuse across all our pages!